src/remote.jsx

```diff
@@ -70,31 +70,35 @@ async function initalizeRemoteEditor() {
  *
  * @param {string} html The HTML content to parse for assets.
  */
-async function loadAssets(html) {
-	const doc = new window.DOMParser().parseFromString(html, 'text/html');
+async function loadAssets( html ) {
+	const doc = new window.DOMParser().parseFromString( html, 'text/html' );
 
 	const newAssets = Array.from(
-		doc.querySelectorAll('link[rel="stylesheet"],script')
-	).filter((asset) => asset.id && !excludedScripts.test(asset.src));
+		doc.querySelectorAll( 'link[rel="stylesheet"],script' )
+	).filter( ( asset ) => asset.id && ! excludedScripts.test( asset.src ) );
 
 	/*
 	 * Load each asset in order, as they may depend upon an earlier loaded script.
 	 * Stylesheets and Inline Scripts will resolve immediately upon insertion.
 	 */
-	for (const newAsset of newAssets) {
-		await loadAsset(newAsset);
+	for ( const newAsset of newAssets ) {
+		await loadAsset( newAsset );
 	}
 }
 
 // Discard remote copies of localy-sourced Gutenberg packages to avoid conflicts
-const localGutenbergPackages = ['api-fetch'];
+const localGutenbergPackages = [ 'api-fetch' ];
 const excludedScripts = new RegExp(
 	localGutenbergPackages
-		.flatMap((script) => [
-			`wp-content/plugins/gutenberg/build/${script.replace(/\//g, '\\/')}\\b`,
-			`wp-includes/js/dist/${script.replace(/\//g, '\\/')}\\b`,
-		])
-		.join('|')
+		.flatMap( ( script ) => [
+			`wp-content/plugins/gutenberg/build/${ script
+				.replace( /\\/g, '\\\\' )
+				.replace( /\//g, '\\/' ) }\\b`,
+			`wp-includes/js/dist/${ script
+				.replace( /\\/g, '\\\\' )
+				.replace( /\//g, '\\/' ) }\\b`,
+		] )
```

Comment:
Added backslash escaping to address [security alerts](https://github.com/wordpress-mobile/GutenbergKit/pull/101#pullrequestreview-2619498670).